luminaire/utils/validation.py

```diff
@@ -0,0 +1,22 @@
+import numpy as np
+import numbers
+
+def check_random_state(seed):
+    """Turn seed into a np.random.RandomState instance
+    Parameters
```

Comment:
You would need to follow the `param`, `return` and `rtype` standard for function comment | [example](https://github.com/zillow/luminaire/blob/master/luminaire/optimization/hyperparameter_optimization.py#L55-L59)